std/algorithm/sorting.d

```diff
@@ -1063,16 +1063,14 @@ sort(alias less = "a < b", SwapStrategy ss = SwapStrategy.unstable,
 {
     int[] array = [ 1, 2, 3, 4 ];
     // sort in descending order
-    sort!("a > b")(array);
-    assert(array == [ 4, 3, 2, 1 ]);
+    assert(array.sort!("a > b").release == [ 4, 3, 2, 1 ]);
```

Comment:
I think the whole point was showing off that the original array is sorted in-place